### PR TITLE
doc(release note): complete 18.10.4 release note

### PR DESCRIPTION
**Pull request by adr-mo**
<h1> Pull Request Template </h1>

<h2> Description </h2>

Release note in order to start validation of the 18.10.4

Fixes # (issue) **None**

<h2> Type of change </h2>

- [ ] Updating documentation (missing information, typo...)

<h2> Target serie </h2>

- [x] 18.10.x
- [x] 19.04.x (master)

<h2> How this pull request can be tested ? </h2>

Display it on GitHub and during the upgrade process.

### CODE REVIEW

File: doc/en/release_notes/centreon-18.10/centreon-18.10.4.rst [hunk 1]
```diff
@@ -5,5 +5,36 @@ Centreon Web 18.10.4
 Enhancements
 ============
 
+* [API] API for commands arguments descriptions (PR/#7196)
+* [API] Add showinstance CLAPI command to Host (PR/#7199)
+* [API] Acknowledge resources using the API (Issue/#6068 - PR/#7187)
+* [UI] Improve host template selection by remplacing simple select with multi-select (##7208)
```
Comment (lpinsivy) [inline doc/en/release_notes/centreon-18.10/centreon-18.10.4.rst]: Remove one #

File: doc/en/release_notes/centreon-18.10/centreon-18.10.4.rst [hunk 2]
```diff
@@ -5,5 +5,36 @@ Centreon Web 18.10.4
...
+* [UI] Indent third level menu (PR/#7251)
+
 Bug Fixes
 =========
+
+* [Centcore] Allow to set illegal characters for centcore (PR/#7206)
```
Comment (lpinsivy) [inline doc/en/release_notes/centreon-18.10/centreon-18.10.4.rst]: should be enhancement

File: doc/fr/release_notes/centreon-18.10/centreon-18.10.4.rst [hunk 3]
```diff
@@ -5,5 +5,36 @@ Centreon Web 18.10.4
 Enhancements
 ============
 
+* [API] API for commands arguments descriptions (PR/#7196)
+* [API] Add showinstance CLAPI command to Host (PR/#7199)
+* [API] Acknowledge resources using the API (Issue/#6068 - PR/#7187)
+* [UI] Improve host template selection by remplacing simple select with multi-select (##7208)
```
Comment (lpinsivy) [inline doc/fr/release_notes/centreon-18.10/centreon-18.10.4.rst]: Remove one #

File: doc/fr/release_notes/centreon-18.10/centreon-18.10.4.rst [hunk 4]
```diff
@@ -5,5 +5,36 @@ Centreon Web 18.10.4
...
+* [UI] Indent third level menu (PR/#7251)
+
 Bug Fixes
 =========
+
+* [Centcore] Allow to set illegal characters for centcore (PR/#7206)
```
Comment (lpinsivy) [inline doc/fr/release_notes/centreon-18.10/centreon-18.10.4.rst]: should be enhancement

**Comment by sc979** [comment]
please add the 7160 ;)

**Comment by adr-mo** [comment]
@sc979 #7160 added to the release note

**Pull request merged**